torch/jit/__init__.py

```diff
@@ -2098,6 +2098,12 @@ def _get_script_class(name):
         return None
     return _script_classes[name]
 
+def _get_module_jit_type(module):
```

Comment:
Is it possible to do something like
```
auto concrete_type = py::getattr(input, "_concrete_type");
auto jit_type = py::getattr(input, "_jit_type");
```

If so, I can get rid of this and do the lookup of the JIT type in C++ in `pybind_utils.h` above.